src/main/java/de/pi3g/pi/oled/OLEDDisplay.java

```diff
@@ -73,6 +73,19 @@
  */
 public class OLEDDisplay {
 
+    /**
+     * Used to specify the orientation of a display.
+     * This allows mounting the oled display upright or flipping it.<br>

```

Comment:
Please don't use `<br>` in javadoc (see https://stackoverflow.com/questions/5260368/which-tag-should-be-used-as-paragraph-separator-in-javadoc)